icons/monitor-down.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "tv",
+    "screen",
+    "display",
+    "desktop",
+    "download",
+    "clone",
+    "version control",
+    "git"
```

Comment:
I like the icon in general, but I'm not sure about these tags, I don't think I've ever seen this type of icon being used for git clone. 🤔

Reply:
Hmm, I was going off this…

<img width="342" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/233833953-5b25475a-cd3f-41d7-9ef6-a11d3b5c8f75.png">

Reply:
I think that's maybe more about GitHub _Desktop_ and less about `git clone`. 😄 

Same with the other "clone" icon in this dropdown, which stands in for "git clone" _using CLI_:

![image](https://user-images.githubusercontent.com/17746067/233834070-9e3bf4e1-de50-442e-a46a-dde3b57649fe.png)